cpg_utils/cromwell.py

```diff
@@ -386,6 +386,9 @@ def watch_workflow(  # noqa: C901
     )
     from cpg_utils.cromwell_model import WorkflowMetadataModel
 
+    # ensure logging info statements are actually printed
+    logging.basicConfig(level=logging.INFO)
```

Comment:
This will apply to all loggers, could we potentially revert your previous change, to create an info logger just for this file?

Reply:
It does, but this method is executed as a distinct script in a separate batch job so there is only one logger. I'll add the same logging setup as before, but `__file__` is not a meaningful name here, as we're creating `script.py`

Reply:
I've re-added the logger instance generation, using the name `cromwell_watcher`